examples/flows/standard/multi_intent_conversational_language_understanding/README.md

```diff
@@ -0,0 +1,46 @@
+# Multi Intent Conversational Language Understanding
+
+A flow that can be used to determine multiple intents in a user query leveraging an LLM with Conversatioanal Language Understanding. 
+
+This sample flow utilizes Azure AI Language's Conversational Language Understanding to perform various analyses on text or documents. It performs:
+
+- Breakdown of compound multi intent user queries into single user queries using an LLM.
+- [Conversational Language Understanding](https://learn.microsoft.com/en-us/azure/ai-services/language-service/conversational-language-understanding/overview) on each of those single user queries.
+
+See the [promptflow-azure-ai-language](https://github.com/microsoft/promptflow/blob/main/docs/reference/tools-reference/azure_ai_language_tool.md) tool package reference documentation for further information. 
+
+Tools used in this flow:
+- `LLM` tool
+- `conversational_language_understanding` tool from the `promptflow-azure-ai-language` package
+
+Connections used in this flow:
+- `Custom` connection
+
+## Prerequisites
+Install promptflow sdk and other dependencies:
+```
+pip install -r requirements.txt
+```
+
+## Setup connection
+Prepare your [Azure AI Language Resource](https://azure.microsoft.com/en-us/products/ai-services/ai-language) first, and [create a Language Resource](https://portal.azure.com/#create/Microsoft.CognitiveServicesTextAnalytics) if necessary. Import the accompanying MediaPlayer.json into a CLU app, train the app and deploy. From your Language Resource, obtain its `api_key` and `endpoint`.
+
+Create a connection to your Language Resource. The connection uses the `CustomConnection` schema:
+```
+# Override keys with --set to avoid yaml file changes
+pf connection create -f ../../../connections/custom.yml --set secrets.api_key=<your_api_key> configs.endpoint=<your_endpoint> name=azure_ai_language_connection
```

Comment:
maybe use a new connection yaml file, as the one referenced here has unexpected fields.